packages/nextra-theme-docs/css/styles.css

```diff
@@ -34,6 +34,19 @@ summary {
   @apply nx-rounded;
 }
 
+.nextra-content {
+  @apply nx-text-slate-700 dark:nx-text-slate-200;
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    @apply nx-text-slate-900 dark:nx-text-slate-100;
+  }
+}
```

Comment:
I think our current content color has too much contrast (#000/#fff), adjusting them slightly will make a huge difference. Many well-designed docs do that.